src/auth/auth-views.controller.ts

```diff
@@ -60,12 +60,13 @@ export class AuthViewsController {
 	@ApiOkResponse({
 		description: 'Render successful.',
 	})
+	@JwtOrUnrestrictedAuth()
 	async renderRegister(
 		@AuthUser() user: User | undefined,
 		@Res() res: Response,
 	): Promise<void | Record<string, string>> {
 		if (user?.id) {
-			return res.redirect('/');
+			return res.redirect('/spots');
```

Comment:
The condition did not work because there was no user without the 	"@JwtOrUnrestrictedAuth()" decorator.

When connected, the user is redirected to the spots page, "yey" ? or "ney"

Reply:
Good catch! I would still redirect the user to the index. They can then decide where to go afterward.

Reply:
Done!